src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -60,9 +58,8 @@ public List<List<AudioFileInfo>> processAudioForDownload(Map<MediaEntity, Path>
 
         List<List<AudioFileInfo>> finalisedAudioSessions = new ArrayList<>();
         for (List<AudioFileInfo> audioSession : groupedAudioSessions) {
-            List<AudioFileInfo> concatenatedAudios = concatenateByChannel(audioSession);
             List<AudioFileInfo> trimmedAudios = trimAllToPeriod(
-                concatenatedAudios,
+                audioSession,
```

Comment:
I don't think we need to worry about the `download` as it will just include all the media within the requested timeframe (trimmed).  There is no concatenation involved now for download.